src/server/pps/persist/server/rethink_api_server.go

```diff
@@ -320,6 +319,41 @@ func (a *rethinkAPIServer) DeletePipelineInfo(ctx context.Context, request *ppsc
 	return google_protobuf.EmptyInstance, nil
 }
 
+func (a *rethinkAPIServer) ShardStart(ctx context.Context, request *ppsclient.Job) (response *persist.JobInfo, retErr error) {
+	defer func(start time.Time) { a.Log(request, response, retErr, time.Since(start)) }(time.Now())
+	return a.shardOp(ctx, request, "ShardsStarted")
+}
+
+func (a *rethinkAPIServer) ShardSucceed(ctx context.Context, request *ppsclient.Job) (response *persist.JobInfo, retErr error) {
+	defer func(start time.Time) { a.Log(request, response, retErr, time.Since(start)) }(time.Now())
+	return a.shardOp(ctx, request, "ShardsSucceeded")
+}
+
+func (a *rethinkAPIServer) ShardFail(ctx context.Context, request *ppsclient.Job) (response *persist.JobInfo, retErr error) {
+	defer func(start time.Time) { a.Log(request, response, retErr, time.Since(start)) }(time.Now())
+	return a.shardOp(ctx, request, "ShardsFailed")
+}
```

Comment:
Could we switch the names on this to VerbNoun rather than NounVerb to match other functions?